test/chainlinkAggregator.test.ts

```diff
@@ -0,0 +1,128 @@
+import { Signer } from '@ethersproject/abstract-signer'
+import chai from 'chai'
+import chaiAsPromised from 'chai-as-promised'
+import hre from 'hardhat'
+import { ChainlinkAggregator } from '../types/typechain'
+import { getMarket } from '../tasks'
+import { Address } from '../types/custom/config-types'
+import { ETH_ADDRESS } from '../consts'
+
+chai.should()
+chai.use(chaiAsPromised)
+
+const { deployments, ethers, contracts, getNamedSigner } = hre
+
+describe('Chainlink Aggregator', async () => {
+  let chainlinkAggregator: ChainlinkAggregator
+  let deployer: Signer
+  let sourceTokenAddress: Address
+  let destinationTokenAddress: Address
+  let aggregatorAddressToAdd: Address
+
+  // Setup for global tests
+  beforeEach(async () => {
+    deployer = await getNamedSigner('deployer')
+  })
+
+  describe('Add an additional chainlink aggregator', () => {
```

Comment:
on the second nested `describe` lets start using the function name we are testing as the description
I.e. `describe('add', () => {})`

Reply:
renamed